### PR TITLE
Practice 02

### PR DESCRIPTION
**Pull request by minhng271**
Target:
- Apply knowledge to responsive practice one design
- Used media queries for popular screen size
- Upgrade the existing practice to responsive
- Work fine on Chrome browser latest version

### CODE REVIEW

File: html/practice-two/.gitignore [hunk 1]
```diff
@@ -0,0 +1,4 @@
+.parcel-cache
+dist
+node_modules
+.husky
```
Comment (meovan07) [inline html/practice-two/.gitignore:4]: Missing line end of file

File: html/practice-two/README.md [hunk 2]
```diff
@@ -1,23 +1,54 @@
...
+   $ cd .\html\practice-two\
+   ```
+- Install npm 
+   ```
+   npm install
+   ```
+- Run Project
+   ```
+   npm start
+   ```
```
Comment (meovan07) [inline html/practice-two/README.md:54]: Please re-check your README, why do I have to clone branch practice 1 and then have to checkout practice 2.
And what I do after using `npm start` to run project

File: html/practice-two/src/styles/layout/header.css [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+  }
+
+  .header .menu {
+    display: flex;
+  }
+}
+
+@media screen and (min-width: 1440px) {
+  .header {
+    max-width: 1440px;
```
Comment (meovan07) [inline html/practice-two/src/styles/layout/header.css:24]: I see you use max-width: 1440px a lot, you can make the div container and then responsive for it.

**Comment by meovan07** [comment]
The project is currently not working well on the 375px mobile screen, please fix it.

**Comment by myle-agilityio** [comment]
Lease fix this UI bug, here should have a gap between 2 column
image

**Comment by meovan07** [comment]
The hero image ratio on the tablet screen doesn't seem right. Please check and fix it.

**Comment by hamyy2912** [comment]
Lease fix this UI bug, here should have a gap between 2 column
image

**Comment by meovan07** [comment]
Please fix the UI bug, in how-it-work section and feature section the space on 2 sides is too big in PC view 1376px
image

**Comment by hamyy2912** [comment]
UI Bug:
- Design:<br>image
- Result:<br>image

**Comment by hamyy2912** [comment]
Please try to you svg image for icons and vector images

**Comment by hamyy2912** [comment]
Your footer don't have background color. Please check design again
image

**Comment by hamyy2912** [comment]
I saw you use `max-height` for this card:<br>
image

What if the text in this card is extended like above? The text content will be overfloating from the border, right? So I think you should not use `max-height` here.
Please check again in the other places where you use max/min width and max/min height, I saw you use them in a lot of places.

**Comment by minhng271** [comment]
> The hero image ratio on the tablet screen doesn't seem right. Please check and fix it. image

image
i fixed it using ` aspect-ratio : 16/9; `

**Comment by minhng271** [comment]
> i fixed it using `aspect-ratio : 16/9;`

**Comment by minhng271** [comment]
> Lease fix this UI bug, here should have a gap between 2 column image

image
I fixed it using ` margin-right: 1rem `

**Pull request closed without merging**